from justin.shared.exceptions.justin_error import JustinError


class ExtractorError(JustinError):
    pass
